Gruntfile.js

```diff
@@ -1421,6 +1421,11 @@ module.exports = function (grunt) {
 							"src": "src/css/profile/mobile/theme-changeable/images/3_Controllers/Radio/",
 							"dest": "dist/mobile/theme/changeable/images/3_Controllers/Radio/sprites/",
 							"fileMask": "_[0-9]+\\.svg$"
+						},
+						{
+							"src": "src/css/profile/mobile/theme-changeable/images/0_Round_corner/",
```

Comment:
For implement the round corner, if we use the both svg and shape(css), the corner line is not clear.
So as much as possible, we want to implement it without the svg, just css shape.
If there is not needed svg  file for corner line, UX guide will delete the method of using svg in UX Guide.

Reply:
If we cannot use svg file as mask in footer then the only way is add "flex" style to page and page-content.
Height of "ui-content" has to strict even when AppBar will be during animation of expand. This guarantees that bottom corners will be always visible.